exercises/practice/ocr-numbers/.meta/config.json

```diff
@@ -1,6 +1,29 @@
 {
   "blurb": "Given a 3 x 4 grid of pipes, underscores, and spaces, determine which number is represented, or whether it is garbled.",
   "authors": [],
```

Comment:
another empty one, added in https://github.com/exercism/elixir/commit/93023194ccf7450438df8652c051a5622027a13e by Brian Carrigan